src/NavAbilitySDK.jl

```diff
@@ -73,7 +74,7 @@ include("./navability/services/StandardAPI.jl")
 include("./navability/services/DataBlobs.jl")
 include("./navability/services/Session.jl")
 export getVariable, getVariables, listVariables, ls
-export addVariable, addPackedVariable
+export addVariable, addPackedVariable, addPackedVariableNew, updatePackedVariableNew
```

Comment:
It's all good, but why add a new endpoint. The goal is to deprecate the old endpoint, so SDK should only call the new and remove old references.

I would have changed the old implementation to addPackedVariableOld and pointed existing function to the new endpoint. This moves the needle toward deprecation but updating all existing users of the SDK to new versus having two forks we now need to maintain.

Reply:
Fair, I didn't know whether how we were planning on deprecating the older calls. If that works then yes it's better.

Reply:
I mean, if the signatures are the same, then we don't even need to deprecate the old ones.

Reply:
I believe in SDK there should be no need to deprecate.. just replace. In cloud I run both until all sdks are sorted.

Reply:
I was suggesting old just in case u want it in the event something was acting odd. We know the old is stable.

Reply:
Went with a middle line approach: new functions are there, but standard API uses old function, we can change that as we test it out.